CMakeLists.txt

```diff
@@ -336,6 +336,35 @@ endif (CMAKE_DEBUG_INCLUDES_LDFLAGS)
 set (CPACK_OUTPUT_FILE_PREFIX packages)
 set (CPACK_PACKAGE_NAME "${PROJECT_PACKAGE}")
 set (CPACK_PACKAGE_DESCRIPTION_SUMMARY "Xournal++ - Open source hand note-taking program")
+set (CPACK_DEBIAN_PACKAGE_DESCRIPTION 
```

Comment:
Is it possible to outsource this into a different file? I'm not quit happy to add this into the main CMakeLists.

Reply:
I will try to do it

Reply:
Where should be this description placed?

Reply:
Best would be to place this file somewhere with the docs.

Reply:
Done
Now full package description is stored in `debian/package_description`